promela-mode.el

```diff
@@ -437,8 +437,9 @@ If (match-beginning 2) is non-nil, the item is followed by a `value'."
 
 
 ;; -------------------------------------------------------------------------
+;;;###autoload (add-to-list 'auto-mode-alist '("\\.\\(promela\\|spin\\|pml\\)\\'" . promela-mode))
```

Comment:
I think there should be a line break between the autoload cookie and the `add-to-list` form